Sources/Runner/Commands/Runner.swift

```diff
@@ -176,3 +178,32 @@ func runDanger(logger: Logger) throws {
     // Return the same error code as the compilation
     exit(proc.terminationStatus)
 }
+
+private func bootstrapArgs() -> [String] {
+    #if !os(Linux)
+    return []
+    #else
+    // The Linux bots require extra arguments in order to locate the corelibs.
+    // We can potentially drop this by installing some stable linux toolchain
+    // after Swift gets ABI and module stability.
+    //
+    // Compute if SwiftPM is bootstrapping.
+    let env = ProcessInfo.processInfo.environment
+    guard env.keys.contains("SWIFTPM_BOOTSTRAP") else { return [] }
+    guard let buildPathStr = env["SWIFTPM_BUILD_DIR"], let buildPath = try? AbsolutePath(validating: buildPathStr) else {
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 193;
-->

- :warning: Line should be 120 characters or less: currently 121 characters (`line_length`)